### DISCUSSION TITLE
How amazing the Better Xcloud is.

### DISCUSSION THREAD

**Discussion by ricardo404** [redphx/better-xcloud#17] [General]
Wow, how I wish I had this FPS, Ping and loss information in-game to have feedback on the network. Thank you for this amazing feature Redphx.

You have done more in such a short time than Microsoft has in these years of Xcloud. Thank you.

May you bring even more features for us.

A trophy for you.

![image](https://github.com/redphx/better-xcloud/assets/35298519/b24b4a9f-3b84-4d39-bc68-a74893579c2c)

**Comment by redphx** [comment]
Thanks for the kind words. Your connection looks a lot better than mine.  

I think it's more difficult for big products like xCloud to implement new features because they have to consider many things & test on many devices first. So don't hate them for that 😁

If you don't mind please tell others about this tool.

**Comment by Haisom** [comment]
OMG I'm just amazed by how great this is!! I hope people can do even more with it in the future <3 Thank you @redphx !!!! You're godly!!

**Reply by Haisom** [reply]
PS.: Please let me know if you need help translating it to other languages. I'm brazilian, so I can help with the translation to brazilian portuguese!

**Reply by redphx** [reply]
Thanks for using. I thought about supporting more languages but it's not my top priority right now.

**Reply by Haisom** [reply]
Do you have any patreon or anything like that so we can support you?

**Reply by redphx** [reply]
You're already supporting me by using it 😁

**Comment by DocS788** [comment]
thanks you mate !! It imrpoves the service and it's not a little thing !

**Reply by redphx** [reply]
Thank you
